docs/source/specifications/projjson.rst

```diff
@@ -71,19 +119,123 @@ The high level objects are:
+  The coordinates are expressed in a unspecified datum, with the longitudes
+  relative to the international reference meridian.
+- ``remarks``: (optional) value of type string with an informative text that does
+  not modify the definining parameters of the object. e.g "Use NTv2 file for better accuracy"
+- ``id`` (mutually exclusive with ``ids``): (optional) Identifier of the object, as defined in :ref:`identifiers`
+- ``ids`` (mutually exclusive with ``id``): (optional) Identifiers of the object, as defined in :ref:`identifiers`
+
+If several extents and scopes apply to an object, the ``scope``, ``area`` and ``bbox``
+members should not be used. Instead a ``usages`` member should be used, whose value
+is an array of objects, each of them accepting ``scope`` and/or ``area`` and/or ``bbox``
+as members. While it is acceptable to use the ``usages`` construct for a single
+usage, it is recommended to avoid it and rather use instead the
+``scope``, ``area`` and ``bbox`` members.
+
+Units
++++++
+
+A unit may be described either as:
+
+- an object with the following members:
+    * ``type``: (required) one of the following types:
+      ``LinearUnit``, ``AngularUnit``, ``ScaleUnit``, ``TimeUnit``, ``ParametricUnit``, ``Unit``
+    * ``name``: (required) string.
+    * ``conversion_factor``: (required in most cases, except in the temporal quantities of http://docs.opengeospatial.org/is/18-010r7/18-010r7.html#42)
+      number that expresses a multiplicative factor to convert from the specified
+      unit to a reference unit, as specified in http://docs.opengeospatial.org/is/18-010r7/18-010r7.html#41
+    * ``id`` or ``ids``: (optional, mutually exclusive)
+
+- a string among the following enumeration: ``metre``, ``degree``, ``unity``
+
+Using a string value, when applicable, is recommended for brevity of
+the object definition.
+
+Omitted units in measured parameters
+++++++++++++++++++++++++++++++++++++
+
+Most numeric parameters should generally be accompanied with the corresponding unit.
+
+For example, for a projection parameter:
+
+.. code-block:: json
+
+    {
+        "name": "False easting",
+        "value": 500000,
+        "unit": "metre"
+    }
+
+or:
+
+.. code-block:: json
+
+    {
+        "name": "False easting",
+        "value": 700000,
+        "unit": {
+          "type": "LinearUnit",
+          "name": "foot",
+          "conversion_factor": 0.3048
+        }
+    }
+
+For the following cases, the unit may be omitted if it is metre:
+``semi_major_axis``, ``semi_minor_axis`` and ``radius`` members of an ellipsoid
+
+For the following cases, the unit may be omitted if it is degree:
+``longitude`` of a prime meridian.
+
 Examples
 --------
 
 GeographicCRS
 +++++++++++++
 
-The following invocation
-
-::
-
-    projinfo EPSG:4326 -o PROJJSON -q
+Using a datum member, implicit prime meridian
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-will output:
+The EPSG:4326 / "WGS 84" geographic CRS can be expressed as
```

Comment:
The output below doesn't have the datum ensemble, so it's not strictly speaking the correct EPSG:4326 projjson? 
(in any case it's not what you currently get from `projinfo EPSG:4326 -o PROJJSON -q`, but you also removed that reference ..)

Reply:
Ah, the section below on datum ensembles clarifies that!

Reply:
> The output below doesn't have the datum ensemble, so it's not strictly speaking the correct EPSG:4326 projjson?

There's no definitive answer to that. The datum ensemble concept was introduced only in the EPSG v10 database and PROJ 8.0. When using earlier versions of both, you'd get EPSG:6326 as a datum, and not a datum ensemble.

Reply:
I've changed the first exemple to be another CRS, so that there's only one version of EPSG:4326, and added a note to the example of EPSG:4326 with datum ensemble mentioning the change over time of the definition